content/blog/2020-03-02-react-v16.13.0.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "React v16.13.0"
+author: [threepointone]
+---
+
+Today we are releasing React 16.13.0. It contains bugfixes and new deprecation warnings to help prepare for a future major release.
+
+### Warnings for some updates during render
+
+A React render function is supposed to be a pure function. We do support a local update during a render to [derive state from props](https://reactjs.org/docs/hooks-faq.html#how-do-i-implement-getderivedstatefromprops). However, this only works on the same Component/Hook. If you call it during a render on a different component, this will now issue a warning.
```

Comment:
>A React render function is supposed to be a pure function. 

This might invite purists to debate Hooks.

Maybe something like:

>A React component should not cause side effects in other components during rendering. 

?